static/app/components/replays/walker/splitCrumbs.tsx

```diff
@@ -134,10 +106,14 @@ function SummarySegment({
     </ScrollingList>
   );
 
+  const label =
+    crumbs.length === 1
+      ? getUrl(first(crumbs))
```

Comment:
you checked the length
```suggestion
      ? getUrl(crumbs)
```

Reply:
You can index access though and drop the lodash dep